src/source/Rtcp/RtcpPacket.c

```diff
@@ -1,31 +1,90 @@
 #define LOG_CLASS "RtcpPacket"
 
 #include "../Include_i.h"
+#include "kvsrtcp/rtcp_api.h"
+
+static RTCP_PACKET_TYPE getStandardRtcpPacketType(RtcpPacketType_t packetType)
+{
+    RTCP_PACKET_TYPE ret = 0;
```

Comment:
should n't this be     RTCP_PACKET_TYPE ret = RTCP_PACKET_TYPE_UNKNOWN; ?

Reply:
RTCP_PACKET_UNKNOWN is also zero, but let me update it to the correct value of RTCP_PACKET_TYPE. Nice catch.